continuous_integration/recipes/distributed/meta.yaml

```diff
@@ -44,7 +44,8 @@ outputs:
       script: >
         python -m pip install . -vv --no-deps
         --install-option="--with-cython=profile"  # [cython_enabled]
-      track_features: {{ "cythonized-scheduler" if cython_enabled else "" }}
+      track_features: "[cythonized-scheduler]"    # [cython_enabled]
```

Comment:
Looks like this is renaming the feature by adding `[...]` as well. Is this intended? Or should we keep the same name here?

Reply:
Right. Would suggest reformatting this a bit for clarity. Included the suggestion below.